reader_test.go

```diff
@@ -1883,18 +1895,24 @@ func TestCategoryAssignment(t *testing.T) {
 		t.Fatal(err)
 	}
 
-	categoriesByDiscretionaryData := map[string]string{}
+	categoriesByAccountNumber := map[string]string{}
```

Comment:
switched this test to look up entries by account number, since IAT entries don't have discretionary data